packages/ui/Layout/Cell.js

```diff
@@ -58,6 +59,14 @@ const CellBase = kind({
 		 */
 		component:  PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
 
+		/**
+		 * A ref instance provided by {@link ui/ForwardRef.ForwardRef}.
+		 *
+		 * @type {Function}
+		 * @private
+		 */
+		forwardedRef: PropTypes.func,
```

Comment:
I think that in general we'll want to mark these as `@public` particularly when we export the *Base component. Along that line, I'd like to adopt a more descriptive name to indicate to what you are gaining a reference when using the prop.

What do you think of `componentRef` to align it with the `component` prop?

Reply:
This was marked as private because it's being provided internally by our ForwardRef HOC, which does (or at least should) have a public `ref` method.

I think like `componentRef`, though I'm only hesitant because of my comment above.

Reply:
I was concerned that an author using `CellBase` wouldn't be able to capture a ref because `forwardedRef` stomps on it. However, looking at the implementation again, it looks like an author-provided `ref` prop directly to `CellBase` would override the undefined `forwardedRef`.

Given that, keeping it `@private` makes sense as it makes the `ref`s behave seamlessly in both cases. I do still prefer `componentRef` to `forwardedRef` fwiw.